src/main.cpp

```diff
@@ -140,6 +143,19 @@ int main ( int argc, char** argv )
         }
 
 
+        // Log whole IP-Adresses on server ----------------------------------------------------
+        if ( GetFlagArgument ( argv,
```

Comment:
What does the IP address do for us? It is possible to create a proxy for IP that is anonymized.
Some form of IP address could be needed for diagnostics.

Reply:
> Some form of IP address could be needed for diagnostics.

Agree. The problem was already discussed in #576.

This can of course be removed (and probably will). I personally would like to have this (optional) function in Jamulus since it would then allow the server admin to decide if he wants to hide IPs or not. For me, this might still be a privacy issue.